src/GoodsList.tsx

```diff
@@ -7,8 +7,12 @@ type Props = {
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
+    {goods.map((good) => (
+      <li
+        key={good.id}
+        data-cy="good"
+        style={{ color: `${good.color}` }}
```

Comment:
better use classnames here and add some styles to this classes
```suggestion
        className={good.color}
```